paperqa/agents/tools.py

```diff
@@ -46,6 +47,9 @@ class SharedToolState(BaseModel):
     docs: Docs
     settings: Settings
     profiler: SimpleProfiler = Field(default_factory=SimpleProfiler)
+    _llm_model: LiteLLMModel | None = None
```

Comment:
Can we make a sentinel `POPULATE_FROM_SETTINGS: ClassVar[None] = None` and use that as the default?

It would be more readable

Reply:
great idea